vignettes/adae.Rmd

```diff
@@ -0,0 +1,325 @@
+---
+title: "R Submissions Working Group Pilot 3: Adverse Events Analysis Dataset"
+output:
+  html_document:
+    df_print: paged
+    toc: yes
+    toc_depth: '2'
+    toc_float: true
+---
+
+```{r, message = FALSE}
+# Initiate start-up file
+source(file.path(rprojroot::find_root("DESCRIPTION"), "inst/startup.R"))
+```
+
+```{r}
+# Working directory requires write permission
+if (file.access(".", 2) != 0) {
+  warning(
+    "The working directory '", normalizePath("."), "' is not writable.\n",
+    "Please change it to a location with write permission."
+  )
+}
+```
+
+```{r setup, message=FALSE}
+# CRAN package, please using install.packages() to install
+library(admiral)
+library(dplyr)
+library(tidyr)
+library(lubridate)
+library(stringr)
+library(xportr)
+library(metacore)
+library(metatools)
+library(haven)
+
+# Propitiatory Package, please refer appendix of ADRG to install - library(pilot3wrappers)
+```
+
+# adae.R
+
+## Step 1: Read in data
+
+```{r}
+# read in AE
+# ----------
+ae <- read_xpt(file.path("/cloud/project/sdtm", "ae.xpt"))
+suppae <- read_xpt(file.path("/cloud/project/sdtm", "suppae.xpt"))
+
+
+# read in ADSL
+# ------------
+adsl <- read_xpt(file.path("/cloud/project/submission", "datasets", "adsl.xpt"))
+
```

Comment:
Is there a way to use `envsetup` to make this not dependent on a particular set as this is specific to RStudio Cloud.

Reply:
Yes. With envsetup you can define a list of search paths and their priority. Let me know if you need help setting it up. I am happy to have a look.